Bot/MessageHandlers/Commands/strike.js

```diff
@@ -0,0 +1,84 @@
+const ROLES = {
+    LUCKY : "902322660262957056"
+    STRIKEONE : "896026304237932565",
+    STRIKETWO : "906695092230832139",
+    BANNED : "614522888372092928",
+}
+
+function getMemberFromMention(mention, channel) {
+	if (mention.startsWith("<@") && mention.endsWith(">")) {
+		mention = mention.slice(2, -1);
+
+		if (mention.startsWith("!")) {
+            mention = mention.slice(1);
+			const foundMember = channel.members.find(
+                (member) => member.user.id === mention
+            );
+            return foundMember;
+		}
+
+        return undefined;
+	}
+}
+
+function addRoleById(msg, target, roleId) {
+    let role = msg.guild.roles.cache.find(r => r.id === roleId);
+    target.roles.add(role)
+        .then(() => {
+            console.log(`Given ${role} to <@${target.user.id}>`);
+        })
+        .catch(console.error);
+}
+
+function removeRoleById(msg, target, roleId) {
+    let role = msg.guild.roles.cache.find(r => r.id === roleId);
+    target.roles.remove(role)
+        .then(() => {
+            console.log(`Given ${role} to <@${target.user.id}>`);
```

Comment:
Do we need this? I think it would already go into the Discord server audit log, would prefer to keep console logs to minimum so that looking at the logs on the server side is quicker

Also the log says `Given` but it's in the *REMOVE*role function